### ISSUE TITLE
Subscription class isn't exported

### ISSUE THREAD

**Issue by amantoux** [jonataslaw/VideoCompress#66]
Unless I am missing something, Subscription is not exported right?
If it is the case, any chances to expose it to API users?

**Comment by themadmrj** [comment]
+1, atm we can't unsubscribe to progress callback.

**Comment by marceloreis13** [comment]
I'm facing the same problem

**Comment by hectorAguero** [comment]
Hi, probably this issue is different, but I can't make work the readme example for subscribe to progress, so for anyone that doesn't understand how to make a workaround for that error in subscription, I made this example based on the example file.

Also, I have the same question than if is neccesary to `unsubscribe` to `compressProgress$`


```dart
import 'dart:async';
import 'dart:io';

import 'package:flutter/material.dart';
import 'package:video_compress/video_compress.dart';
import 'package:image_picker/image_picker.dart';

void main() => runApp(MyApp());

class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      home: MyHomePage(),
    );
  }
}

class MyHomePage extends StatefulWidget {
  @override
  _MyHomePageState createState() => _MyHomePageState();
}

class _MyHomePageState extends State<MyHomePage> {
  String _pathVideo;
  final _streamController = StreamController<int>();

  @override
  void initState() {
    super.initState();
    final _videoCompress = VideoCompress;
    _videoCompress.compressProgress$.subscribe((event) {
      _streamController.add(event.round());
    });
  }

  @override
  void dispose() {
    _streamController.close();
    super.dispose();
  }

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(title: const Text('Video Compress')),
      body: Center(
        child: Column(
          mainAxisAlignment: MainAxisAlignment.center,
          children: <Widget>[
            StreamBuilder<int>(
              stream: _streamController.stream,
              initialData: 0,
              builder: (context, snapshot) {
                if (snapshot.data == 0 && _pathVideo == null)
                  return Text('Select Video...');
                if (snapshot.data == 100) return Text(_pathVideo ?? '');
                return Text('Compressing video ${snapshot.data}%');
              },
            ),
          ],
        ),
      ),
      floatingActionButton: FloatingActionButton(
        onPressed: () async {
          if (_pathVideo != null) await File(_pathVideo).delete();
          setState(() => _pathVideo = null);
          final file =
              await ImagePicker().getVideo(source: ImageSource.gallery);
          if (file != null) {
            _streamController.add(0);
            final info = await VideoCompress.compressVideo(file.path,
                quality: VideoQuality.MediumQuality, includeAudio: true);
            if (info != null) setState(() => _pathVideo = info.path);
          } else {
            _streamController.add(0);
          }
        },
        tooltip: 'Compress Video',
        child: const Icon(Icons.video_library),
      ),
    );
  }
}`

**Comment by hyobbb** [comment]
Same issue. Any notice??

**Comment by jonataslaw** [comment]
Fixed on 2.1.1
